src/main/resources/templates/kotlin-lang/kotlinClassGraphqlRequest.ftl

```diff
@@ -0,0 +1,149 @@
+<#list annotations as annotation>
+@${annotation}
+</#list>
+open class ${className}(private val alias: String?) : GraphQLOperationRequest {
+
+    companion object {
+        const val OPERATION_NAME: String = "${operationName}"
+        val OPERATION_TYPE: GraphQLOperation = GraphQLOperation.${operationType}
+<#if builder>
+
+        fun builder(): Builder = Builder()
+</#if>
+    }
+
+    private val input: MutableMap<String, Any?> = LinkedHashMap()
+
+    constructor(): this(null)
+
+<#if fields?has_content>
+<#list fields as field>
+<#if field.javaDoc?has_content>
+    /**
+<#list field.javaDoc as javaDocLine>
+     * ${javaDocLine}
+</#list>
+     */
+</#if>
+<#if field.deprecated>
+    @java.lang.Deprecated
+</#if>
+    fun set${field.name?cap_first}(${field.name}: ${field.type}) {
+        this.input["${field.originalName}"] = ${field.name}
+    }
+
+</#list>
+</#if>
+    override fun getOperationType(): GraphQLOperation = OPERATION_TYPE
+
+    override fun getOperationName(): String = OPERATION_NAME
+
+    override fun getAlias(): String? = alias ?: OPERATION_NAME
+
+    override fun getInput(): MutableMap<String, Any?> = input
+
+<#if equalsAndHashCode>
+    override fun equals(other: Any?): Boolean {
+        if (this === other) {
+            return true
+        }
+        if (other == null || javaClass != other.javaClass) {
+            return false
+        }
+        val that = other as ${className}
+        return Objects.equals(operationType, that.operationType) &&
+                   Objects.equals(operationName, that.operationName) &&
+                   Objects.equals(input, that.input)
+    }
+
+    override fun hashCode(): Int = Objects.hash(operationType, operationName, input)
+</#if>
+<#if toString>
+
+    override fun toString(): String = Objects.toString(input)
+</#if>
+
+<#if builder>
+    class Builder {
+
+        private var `$alias`: String? = null
+    <#if fields?has_content>
+        <#list fields as field>
+        <#if field.defaultValue?has_content>
+        private var ${field.name}: ${field.type} = ${field.defaultValue}
+        <#else>
+        <#if field.type?ends_with("?")>
+        private var ${field.name}: ${field.type} = null
+        <#else>
+        <#if MapperUtil.isKotlinPrimitive(field.type)>
+        <#assign default = MapperUtil.defaultValueKotlinPrimitive(field.type)/>
+        private var ${field.name}: ${field.type} = default
+        <#else>
+        private lateinit var ${field.name}: ${field.type}
+        </#if>
+        </#if>
+        </#if>
+        </#list>
+    </#if>
+
+        fun alias(alias: String?): Builder {
+            this.`$alias` = alias
+            return this
+        }
+
+    <#if fields?has_content>
+        <#list fields as field>
+            <#if field.javaDoc?has_content>
+        /**
+        <#list field.javaDoc as javaDocLine>
+         * ${javaDocLine}
+        </#list>
+         */
+    </#if>
+    <#if field.deprecated>
+        @java.lang.Deprecated
```

Comment:
I think we should use the Kotlin equivalent here.

Reply:
It's not going to be used for the time being, because kotlin needs some additional parameters

Reply:
I think all you need is a message here. So we could have a default message like "this is deprecated in GraphQL".

But if I remember correctly, the reason for deprecation is also mandatory in the GraphQL schema, right?
Then we should use that same reason message here in Kotlin.

Reply:
At present, it seems that it is not necessary, but I will try the default message

Reply:
It looks like the reason is an optional parameter in the `@deprecated` directive in GraphQL, but there is a default reason in that case, which is "No longer supported". So I think there is always a reason that we could use here.